DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
 \setDocDate{\vcsdate}
 \setDocUpstreamLocation{\url{https://github.com/lsst/lsst-texmf/examples}}
-\author{% `git log --pretty=%an | sort --key=2 | uniq` ?
-  Eric Bellm
-}
-
-% Most recent last
-\setDocChangeRecord{
-\addtohist{}{2018-07-16}{Initial version.}{Belm, Comoretto}
-}
+\author{ Eric Bellm }
 
-\setDocCurator{Eric Bellm}
-\setDocUpstreamLocation{\url{https://github.com/lsst-dm/\lsstDocType-\lsstDocNum}}
-\setDocUpstreamVersion{\vcsrevision}
+\input history_and_info.tex
 
 
 \setDocAbstract{
-This is the test report for \milestoneId{} (\milestoneName{}), an LSST DM level 1 milestone pertaining to the \product{}.
+This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST level 2 milestone pertaining to the \product{} Subsystem.
 }
 
+
 \maketitle
 
 \section{Introduction}
 \label{sect:intro}
 
+
 \subsection{Objectives}
 \label{sect:objectives}
 
-This test activity demonstrates the successful execution of the major components of the Alert Distribution system at the scale of LSST Operations.
+This test activity demonstrates the successful execution of the major
+components of the Alert Distribution system at the scale of LSST
+Operations.\\[2\baselineskip]It will demonstrate that:\\
 
-It will demonstrate that:
 \begin{itemize}
-\item The Alert Distribution payloads can be made available on systems managed by the LSST Data Facility
-\item The Alert Distribution payloads can be executed under the control of the relevant LDF services
-\item All required science data products can be passed through the components of the alert distribution service.
+\tightlist
+\item
+  The Alert Distribution payloads can be made available on systems
+  managed by the LSST Data Facility
+\item
+  The Alert Distribution payloads can be executed under the control of
+  the relevant LDF services
+\item
+  All required science data products can be passed through the
+  components of the alert distribution service.
 \end{itemize}
 
-Note that this test plan does not extend to testing integration of the Alert Distribution System with the Alert Generation Science Pipeline or the LSST Science Platform.
-We also do not test the Alert Database at this time as its functional requirements are not sufficiently specified to enable testing.
+Note that this test plan does not extend to testing integration of the
+Alert Distribution System with the Alert Generation Science Pipeline or
+the LSST Science Platform.\\
+We also do not test the Alert Database at this time as its functional
+requirements are not sufficiently specified to enable
+testing.\\[2\baselineskip]
 
-\subsection{Scope}
-\label{sect:scope}
+\subsection{Scope}\label{scope}
 
-The overall test plan for the LSST Data Management system is described in \citeds{LDM-503}.
+The overall test plan for the LSST Data Management system is described
+in \citeds{LDM-503}.\\
+This document specifically refers to the milestone LDM-503-5, which
+tests the LSST Alert Dis- tribution System.\\
```

Comment:
The LaTeX is generated using pandoc, no control over it.

This text is part of the Objective field in LVV-P1. Is there where it has to be changed.

Reply:
> The LaTeX is generated using pandoc, no control over it.
I don't think this is adequate: if our tools don't work, we need new tools.

Reply:
OK. We can look for different tools, or better use of pandoc. However, the pdf seems to be rendered correctly.

Reply:
I have a couple of thoughts here.

First: if there's a problem with Pandoc, then we should discuss that problem. I'd expect that we'd start by filing a ticket in the LSST Jira system to describe the problem. This particular ticket probably wouldn't be high priority, so I wouldn't expect it to be worked on right away, but we could refer to it here rather than just ignoring the issue. Then ultimately, perhaps, we could file an upstream bug report, or one rainy afternoon somebody on-Project could dive into the source and fix it.

However, it turns out this isn't a problem with Pandoc:
```
$ curl -u swinbank https://jira.lsstcorp.org/rest/atm/1.0/testplan/LVV-P1 | python -m json.tool | grep "Dis\-"
Enter host password for user 'swinbank':
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4891    0  4891    0     0  15189      0 --:--:-- --:--:-- --:--:-- 15189
    "objective": "This test activity demonstrates the successful execution of the major components of the Alert Distribution system at the scale of LSST Operations.<br><br>It will demonstrate that:<br><ul><li>The Alert Distribution payloads can be made available on systems managed by the LSST Data Facility</li><li>The Alert Distribution payloads can be executed under the control of the relevant LDF services</li><li>All required science data products can be passed through the components of the alert distribution service.</li></ul><br>Note that this test plan does not extend to testing integration of the Alert Distribution System with the Alert Generation Science Pipeline or the LSST Science Platform.<br>We also do not test the Alert Database at this time as its functional requirements are not sufficiently specified to enable testing.<br><br><strong>Scope</strong><br><br>The overall test plan for the LSST Data Management system is described in LDM-503.<br>This document specifically refers to the milestone LDM-503-5, which tests the LSST Alert Dis- tribution System.<br>The overall LSST Alert Distribution System test specification is defined in LDM-533.<br><br>This plan is identified in JIRA as LVV-P1. The related test run is LVV-R3 that foresee the execu- tion of the following test cases:<br>LVV-T216(1.0) Installation of the Alert Distribution payloads LVV-T217(1.0) Full Stream Alert Distribution<br>LVV-T218(1.0) Simple Filtering of the LSST Alert Stream<br><br><br>",
```
We should fix this on Jira.